website/docs/r/site_recovery.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "vmc"
+
+page_title: "VMC: vmc_site_recovery"
+sidebar_current: "docs-vmc_site_recovery"
+
+description: |-
+  Provides a resource to activate and deactivate site recovery for an SDDC.
+---
+
+# vmc_site_recovery
+
+Provides a resource to activate and deactivate site recovery for an SDDC.
+~> **Note:** Site recovery resource implicitly depends on SSDC resource creation. SDDC must be provisioned before a site recovery can be activated. For details on how to provision a SDDC refer to the SDDC documentation.
```

Comment:
SDDC documentation can be linked to the existing one https://www.terraform.io/docs/providers/vmc/r/sddc.html

Reply:
I will add this